setup.cfg

```diff
@@ -0,0 +1,4 @@
+[develop]
+script_dir=$base/lib/rqt_gauges_2
+[install]
+install_scripts=$base/lib/rqt_gauges_2
```

Comment:
nit: missing empty line at the end of the file

Reply:
Done.